AssignmentManager/AssignmentMembersApi.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage.Table;
+using StudentsAccessor;
+
+namespace AssignmentsManager
+{
+    public static class AssignmentMembersApi
+    {
+        [FunctionName("GetAssignmentMembers")]
+        public static async Task<IActionResult> GetAssignmentMembers(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = "assignments/{assignmentGuid}/members")] HttpRequest req,
+            [Table("Assignments", Connection = "StorageConnection")] CloudTable assignments,
+            string assignmentGuid,
+            ILogger log)
+        {
+            if (!Guid.TryParse(assignmentGuid, out var queryQuid))
+            {
+                return new BadRequestErrorMessageResult($"Invalid assignment Guid {assignmentGuid}");
+            }
+            var lmsAssignment = await GetLmsAssignment(assignments, queryQuid);
+            if (lmsAssignment == null)
+            {
+                return new NotFoundObjectResult($"Assignment with Guid {assignmentGuid} was not found");
+            }
+            var membershipsManager = new LmsMemberships();
+            var members = await membershipsManager.GetMemberships(lmsAssignment.CustomContextMembershipsUrl,
```

Comment:
It's should call membershipsAccessor not membershipsManager

Reply:
Is it only about naming? call it lmsMembershipsAccessor instead